chapters/api/content/api.md

```diff
@@ -0,0 +1,92 @@
+# Exposing and Querying Smart Contract Data on MultiversX
+
+This lab outlines the architecture and methods for fetching real-time and historical data related to the MultiversX network.
```

Comment:
Can you add an intro here explaining that the principle discussed in this section apply to other blockchains as well, but we'll use MultiversX to demonstrate these principles/approach.